apps/token/src/routes/rewards/home/index.tsx

```diff
@@ -0,0 +1,164 @@
+import './index.scss';
+
+import { useQuery } from '@apollo/client';
+import { Button, Callout, Intent, Splash } from '@vegaprotocol/ui-toolkit';
+import { formatDistance } from 'date-fns';
+// @ts-ignore TODO: check if duration-js has a @types definition
```

Comment:
?

Reply:
This ts-ignore came over from the old repo. There are no types available for duration-js so I've left the ts-ignore in for now